students/rfelts/lesson02/series.py

```diff
@@ -0,0 +1,114 @@
+# Russell Felts
+# Assignment 2 - Fibonacci and Lucas Series
+#
+# Add a function called fibonacci. it should have one parameter, n. and return
+# the nth value in the fibonacci series (starting with zero index).
+#
+# Add a function, lucas, that returns the nth value in the lucas numbers series
+#
+# Add a third function called sum_series with one required parameter and two
+# optional parameters. The required parameter will determine which element in
+# the series to print. The two optional parameters will have default values of
+# 0 and 1 and will determine the first two values for the series to be
+# produced. Calling this function with no optional parameters will produce
+# numbers from the fibonacci series. Calling it with the optional arguments 2
+# and 1 will produce values from the lucas numbers. Other values for the
+# optional parameters will produce other series.
+#
+# Ensure that your functions have a well-formed docstring
+
+
+def fibonacci(n):
+    """
+
+    Return the fibonacci number in the series of the value passed in.
+
+    Parameters:
+    arg1 (int) The desired positon in the fibonacci series
+
+    Returns:
+    int: The nth value in the fibonacci series
+
+    """
+
+    if n > 1:
+        return fibonacci(n - 1) + fibonacci(n - 2)
+    else:
+        return n
+
+
+def lucas(n):
+    """
+
+    Return the lucas number in the series of the value passed in.
+
+    Parameters:
+    arg1 (int) The desired positon in the lucas series
+
+    Returns:
+    int: The nth value in the lucas series
+
+    """
+
+    if n == 0:
+        return 2
+    elif n == 1:
+        return 1
+    else:
+        return lucas(n - 1) + lucas(n - 2)
+
+
+def sum_series(n, seriesA=0, seriesB=1):
```

Comment:
note that per python style guide, var names (and functions) should follow snake_case convention -> `series_a`